### PR TITLE
Fix OffsetManager#seek_to_default

### PR DESCRIPTION
**Pull request by ale7714**
* Add tests to OffsetManager

Follow up to https://github.com/zendesk/ruby-kafka/pull/386, while adding tests to `OffsetManager`  I realized the my change broke `#seek_to_default`  😓.

### CODE REVIEW

File: spec/offset_manager_spec.rb [hunk 1]
```diff
@@ -203,4 +203,29 @@ def partition_offset_info(offset)
...
+      expect(offset).to eq seek_offset
+    end
+  end
+
+  describe "#seek_to_default" do
+    it "delegates to seek_to" do
+      topic = "greetings"
+      partition = 0
+
+      expect(offset_manager).to receive(:seek_to).with(topic, partition, -1)
```
Comment (dasch) [inline spec/offset_manager_spec.rb]: I'd rather avoid setting call expectations – this is easily testable by calling `next_offset_for`.

Comment (ale7714) [inline spec/offset_manager_spec.rb]: @dasch good point! updated the test accordingly

**Comment by dasch** [comment]
:heart:

**Comment by ale7714** [comment]
Not sure that the spec fail relates to my code change

**Comment by dasch** [comment]
Restarting the tests fixed the issue. They're a bit flaky.

**Pull request merged**